### ISSUE TITLE
User is sometimes not set  downloadCallback

### ISSUE THREAD

**Issue by ToyboxLabs** [VeliovGroup/Meteor-Files#692]
I was ovrride the download callback to restrict what users can download the file (and for a few other specific reasons) and some time ago the userId stopped being set. It ended up being because of a change in meteor 

https://github.com/meteor/meteor/commit/79ae1849f5846e207ed4dfa49831ba409252fd03#diff-18cd8c5cbe21efdaf022e77bd5c89a40

Where they switched how Meteor.server.sessions was accessed. So I had to add a line of code
```

const params = this.request.cookies
      const sessionId = String(params.x_mtok);
       
      this.userId  = (Meteor.server.sessions.get(sessionId) && Meteor.server.sessions.get(sessionId).userId) ? Meteor.server.sessions.get(sessionId).userId : null;
```

Just thought i would put this here in case anyone else runs into it, not sure if its the right thing to change the package or not.

**Comment by jankapunkt** [comment]
@ToyboxLabs can you please look at this pull request: #683 
It has been merged already and will be part of the next package release.
Would this fix your issue?

**Comment by dr-dimitru** [comment]
Dear @ToyboxLabs ,

This issue should be fixed in [`v1.11.0`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.11.0), please update to the latest release.
Feel free to close it in case if the issue is solved on your end.

**Comment by dr-dimitru** [comment]
Closed due to silence at issue owner end.
Feel free to reopen it in case if the issue still persists on your end.
